site/plugins/redirects/index.php

```diff
@@ -0,0 +1,42 @@
+<?php
+
+use Kirby\Http\Router;
+
+Kirby::plugin('getkirby/redirects', [
+    'hooks' => [
+            'route:after' => function ($route, $path, $method, $result, $final) {
+                    // only if route didn't match anything and is final route
+                    if ($final === true && empty($result) === true) {
+
+                        // load redirects definition
+                        $root      = kirby()->root('config');
+                        $redirects = require $root . '/redirects.php';
+
+                        // turn into routes array
+                        $routes = array_map(function($from, $to) {
+                            return [
+                                'pattern' => $from,
+                                'action'  => function (...$parameters) use ($to) {
+
+                                    // resolve callback
+                                    if (is_callable($to) === true) {
+                                        $to = call_user_func($to, ...$parameters);
```

Comment:
@distantnative Your plugins are always imaginative & inventive. I learn a lot reading the code. Thank you!

Isn't `$to(...$parameters)` more PHP7-esque than using `call_user_func()`?

Reply:
Thank you, @johannschopplich!